libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -74,7 +74,8 @@ proc init*(_: type[GossipSubParams]): GossipSubParams =
       behaviourPenaltyWeight: -1.0,
       behaviourPenaltyDecay: 0.999,
       disconnectBadPeers: false,
-      enablePX: false
+      enablePX: false,
+      bandwidthEstimateMbps: 100
```

Comment:
0 should disable flood-limiting completely

Reply:
I find it more flexible and easier when API:s use basic units rather than pre-baked multiples - there's more flexibility this way for API users (ie imagine working on an embedded system with more aggressive bandwidth constraints).

This is somewhat different from when working with "normal users" or end-user UX.